libraries/rib-debug-utils/src/main/java/com/badoo/ribs/core/plugin/utils/debug/DebugControls.kt

```diff
@@ -0,0 +1,12 @@
+package com.badoo.ribs.core.plugin.utils.debug
+
+import android.view.View
+import android.view.ViewGroup
+import com.badoo.ribs.core.Rib
+
+abstract class DebugControls<T : Rib>(
+    override val label: String,
```

Comment:
do we need `DebugControls` class? maybe we can create a label parameter in `AbstractDebugControls`?

Reply:
Yep, to disallow overriding the other parameters in the parent, which can only be done by the host class.